drivers/usbhid-ups.h

```diff
@@ -142,6 +143,7 @@ typedef enum {
 	BOOST,		/* SmartBoost */
 	BYPASSAUTO,	/* on automatic bypass */
 	BYPASSMAN,	/* on manual/service bypass */
+	ECOMODE,    /* High Efficiency (aka ECO Mode) */
```

Comment:
I think some other code (nutconf C++? aug lens?) and configs and docs and client bindings would all need to be checked, perhaps updated, for the new `ECO` status keyword (if it is not yet there).

Reply:
Done in later commits